robolectric/src/test/java/org/robolectric/shadows/SQLiteDatabaseTest.java

```diff
@@ -910,4 +915,16 @@ public void shouldGetBlobFromString() {
         data.moveToFirst();
         assertThat(data.getBlob(0)).isEqualTo(values.getAsString("first_column").getBytes());
     }
+
+    /////////////////////
+
+    private SQLiteDatabase openOrCreateDatabase(String name) {
```

Comment:
Could these be created as memory backed databases for faster test performance?

https://developer.android.com/reference/android/database/sqlite/SQLiteDatabase.html#create(android.database.sqlite.SQLiteDatabase.CursorFactory)